tests/test_OSF_API.py

```diff
@@ -0,0 +1,119 @@
+import unittest
+
+import requests
+from config.configurations import OSF_API_GEEF_FOUTEN_URL, OSF_API_GEEF_TOTALEN_UITKOMST_URL
+
+
+class TestOSF_GeefTotalenUitkomstURL(unittest.TestCase):
+    def setUp(self):
+        self.osf_url = OSF_API_GEEF_TOTALEN_UITKOMST_URL
+
+    def test_zonder_bestand_geeft_http500(self):
+        response = requests.post(self.osf_url)
+
+        self.assertEqual(500, response.status_code, "De HTTP statuscode van de response is niet gelijk aan 500.")
```

Comment:
je kunt eventueel doen

```
from http import HTTPStatus
self.assertEqual(HTTPStatus.INTERNAL_SERVER_ERROR, response.status_code, "De HTTP statuscode van de response is niet gelijk aan 500.")
```

Ik weet trouwens niet wat nu met OSF is afgesproken. Ik dacht dat we altijd status 200 (OK) terug zouden krijgen.